### PR TITLE
Fixture fix

### PR DESCRIPTION
**Pull request by Programmer-yar**
This PR is related to issue https://github.com/turian/audio-discrimination-crowdsource/issues/16
Before applying changes in this PR,

1. Go to http://localhost:8000/admin/sites/site/ and delete all existing entries

After applying this PR

- Load fixtures according to instructions in `README.md`
- Set the `DEVELOPMENT_MODE` environment variable according to environment you are using

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,14 @@ mkdir .venv
 python3 -m venv .venv
 source .venv/bin/activate
 pip install -r requirements.txt
+pip install pre-commit
```
Comment (turian) [inline README.md:15]: Put this in requirements.txt

File: django_app/settings.py [hunk 2]
```diff
@@ -85,28 +84,30 @@
...
 ]
 
 # db id of the entry in 'Sites' table
-SITE_ID = 1
-if DEVELOPMENT_MODE:
+if DEVELOPMENT_MODE == "production":
+    SITE_ID = 3
+elif DEVELOPMENT_MODE == "staging":
     SITE_ID = 2
+else:
```
Comment (turian) [inline django_app/settings.py:97]: No. You must check ALL three cases and if it's something different (like "True" or "False" or whatever) you should throw an exception.

File: README.md (hunk 1 above)
Comment (Programmer-yar) [reply README.md:15]: Ok

File: django_app/settings.py (hunk 2 above)
Comment (Programmer-yar) [reply django_app/settings.py:97]: Ok

**Review by turian — APPROVED** [review]

**Pull request merged**